app/services/permission_service.rb

```diff
@@ -30,7 +30,7 @@ def platform_admin_permissions(controller, action)
       return true if controller == 'admin/dashboard' && action.in?(%w(index show))
       return true if controller == 'admin/locations' && action.in?(%w(index show))
       return true if controller == 'users'           && action.in?(%w(show edit update profile))
-      return true if controller == 'stores'          && action.in?(%w(index edit validate))
+      return true if controller == 'stores'          && action.in?(%w(index edit validate show))
```

Comment:
Line is too long. [96/80]<br>Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.